.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx commitlint --edit $1
```

Comment:
```suggestion
npx --no-install commitlint --edit $1
```
Minor improvement. I just verified this works on Windows, but downloads `commitlint` every time I commit, even if it's already installed locally as a dev dependency.

Reply:
Sorry for that, i am working on ubuntu